src/pudl/metadata/fields.py

```diff
@@ -2059,7 +2059,17 @@
     },
     "purchased_mwh": {
         "type": "number",
-        "description": "Megawatt-hours shown on bills rendered to the respondent.",
+        "description": "Megawatt-hours shown on bills rendered to the respondent. Includes both electricity purchased for storage and non-storage purposes, which were lumped together prior to 2021.",
+        "unit": "MWh",
+    },
+    "purchased_other_than_storage_mwh": {
+        "type": "number",
+        "description": "Number of megawatt hours purchased for other than energy storage during the period.",
```

Comment:
"Number of megawatt hours purchased for non-storage purposes during the period" would be clearer here.

Reply:
I cut-and-pasted these from the FERC Taxonomy but I agree they read weird.